identity.go

```diff
@@ -326,7 +383,10 @@ func (id *ID) SetServerCert(pem string) error {
 		id.Config.ServerCert = StoragePem + ":" + pem
 		return fmt.Errorf("could not save client certificate, location scheme not supported for saving (%s): \n %s", id.Config.Cert, pem)
 	case StorageFile, "":
-		f, err := os.OpenFile(id.Config.ServerCert, os.O_RDWR, 0664)
+
+		absPath, err := filepath.Abs(id.Config.ServerCert)
```

Comment:
same here fwiw, err isn't checked. just making sure it's intentional

Reply:
fixed